src/colorizer/Dataset.ts

```diff
@@ -66,6 +80,18 @@ export default class Dataset {
     this.outliers = (await this.featureLoader.load(url)).tex;
   }
 
+  private async loadTracks(): Promise<void> {
+    if (!this.tracksFile) {
+      return;
+    }
+    const url = this.resolveUrl(this.tracksFile);
+    const { trackIds, times } = await this.tracksLoader.load(url);
+    this.trackIds = trackIds;
+    this.times = times;
+    console.log(this.times);
+    console.log(this.trackIds);
```

Comment:
Other console.logs are useful to debug this incomplete feature, but these raw data ones probably shouldn't remain